store.go

```diff
@@ -365,6 +365,8 @@ type Store interface {
 	// PrepareStagedLayer applies a diff to a layer.
 	// It is the caller responsibility to clean the staging directory if it is not
 	// successfully applied with ApplyStagedLayer.
+	// The caller must ensure ApplyStagedLayer or CleanupStagedLayer is called eventually with
```

Comment:
nit: we can start using references to other functions/methods, i.e.
```suggestion
// The caller must ensure [Store.ApplyStagedLayer] or [Store.CleanupStagedLayer] is called eventually with
```

Reply:
thanks, I've fixed only this instance for now and the new comments I've added